explore/src/main/scala/explore/constraints/ConstraintsSummaryTable.scala

```diff
@@ -225,7 +225,7 @@ object ConstraintsSummaryTable extends TableHooks:
       )
       // Memo rows
       .useMemoBy((props, _, _) => props.constraintList)((_, _, _) =>
-        _.map(ConstraintGroup.fromTuple).toList
+        _.map(ConstraintGroup.fromTuple).toList.sortBy(_.constraintSet.summaryString)
```

Comment:
Keeps table sorting stable (as long as the user didn't select another sort criterion).